src/bin/add_filter.rs

```diff
@@ -38,11 +35,12 @@ async fn main() {
         }
     };
 
-    // create a bson document with filter_id, active, catalog, permissions
+    // Create a bson document with id, active, catalog, permissions
     // group_id, and a fv array with one doc that has a fid field and a pipeline field
+    let filter_id: String = uuid::Uuid::new_v4().to_string();
 
     let filter = doc! {
-        "filter_id": filter_id,
+        "id": filter_id.clone(),
```

Comment:
Something else strange about this script is that we're going to allow adding filters that have no users attached to them. Maybe that's okay, i.e., we don't need to know where the filters came from so long as they're there and they work? I guess that makes sense if we have one output stream per survey, but then anyone who subscribes to BOOM gets the results of everyone else's filters?

Reply:
This script really is just here for development. We used to not have an API, so that script was the only way to add filters without dealing with mongo directly. We could just make it so that this script gets the admin user assigned to it (which rn is generated when you run the API, but it should really just be part of some generic DB crate/module that both api and "nightly processing" stuff use).